### ISSUE TITLE
Any validation set used?

### ISSUE THREAD

**Issue by ovshake** [jonmun/MM-SADA_Domain_Adaptation_Splits#3]
Hi, 
Thank you for the wonderful work! Is there any validation set present for the domains to understand which checkpoint to use for testing? 

Thanks

**Comment by dimadamen** [comment]
Thanks for your interest in our work. As we specify in Section 4.2:
"For all results, we report the top-1 target accuracy averaged over the last 9 epochs of training, for robustness."
Further details are given in the section before on the 9K steps for training and the learning rate.
Hope this answers your question.
